isis/src/base/objs/NthOrderPolynomial/unitTest.cpp

```diff
@@ -0,0 +1,56 @@
+#include "NthOrderPolynomial.h"
```

Comment:
If these are new unit tests, why not make them gtests? You can apply the same comment to any other unittests in here.

Reply:
@Kelvinrr Agreed! I ran out of time toward the end of the sprint (i.e. Friday) and just made them old-style unit tests because I could write them much more quickly. We have "convert to gtests" on our todo list for the next EIS sprint. The `NthOrderPolynomial` one would be easy to convert and I could probably do it as a repose to this review, but all the camera model gtests were more difficult because they required a spiceinited cube -- I was in the process of trying to figure out how to add the various spice tables to a test cube in a gtest and came up with something that worked for one of the tables, but was long/messy/lots of code duplication / not ideal, and I didn't have time to improve it enough to use it. Definitely open to suggestions here if you have any ideas!

Reply:
It's probably worth, in this PR, to simply convert the polynomial test and leave camera models alone. We can figure out how to exactly create cameras and cubes for testing later.